src/feedback.tsx

```diff
@@ -78,20 +88,33 @@ const renderFeedbackBody = (
         ) : (
             description
         );
+    const handleAnimationEnd = () => {
+        setShouldRead(true);
+        divRef.current?.focus();
+        setHideOthers(false);
+    };
+
     return (
         <Stack space={24}>
             <div className={styles.assetContainer} data-testid="icon">
                 {asset}
             </div>
             <Stack space={16} className={classnames(styles.feedbackData)}>
-                <div className={classnames(animateText && styles.feedbackTextAppearFast)} data-testid="title">
+                <div
+                    ref={divRef}
+                    className={classnames(animateText && styles.feedbackTextAppearFast)}
+                    data-testid="title"
+                    onAnimationEnd={handleAnimationEnd}
+                    tabIndex={-1}
+                >
                     <Text6 as="h1">{title}</Text6>
                 </div>
 
                 {normalizedDescription && (
                     <div
                         className={classnames(animateText && styles.feedbackTextAppearMedium)}
                         data-testid="description"
+                        aria-hidden={hideOthers}
```

Comment:
why do these divs need to be hiden?they are also animated

Reply:
they are hidden because if not, the screen reader reads then before the title